### ISSUE TITLE
Ticket and Registration Proposal

### ISSUE THREAD

**Issue by irabinovitch** [socallinuxexpo/SCALE-Planning#192]
Lei has offered to refactor our ticket options to simplify them and ensure they are easily understood by attendees. 

Issue we'd like to solve:
- Make it clear what each ticket includes access to.
- Reduce total number of tickets if possible.
- Simplify cash payments by picking pricing in $20 increments.

**Comment by irabinovitch** [comment]
From @leizleiz:

I'd like to propose the following ticket changes for SCALE 15X. Some of this was already discussed, and some bits are being proposed just now:

- Expo Pass: $20
- Foo Limited Pass: $20 + Foo event fee of ($0 to whatever)
- Bar Limited Pass: $20 + Bar event fee of ($0 to whatever)
- SCALE Pass (early): $75
- SCALE Pass (normal): $85
- SCALE Pass (late): $90
- SCALE Pass (cash): $100
- SCALE Pass (sunday): $40
- SCALE Pass (sunday/cash): $40
- Supporter Pass: $100
- S.T.A.R. Qux Pass: $75 + Qux event fee
- S.T.A.R. Baz Pass: $75 + Baz event fee

To explain in detail on terminology:

- Hosted event - any event co-hosted as SCALE, not part of SCALE.
- Limited event - a hosted event that requires pre-registration, may
  have limit on number of tickets available. As opposed to hosted events
  that are open to everyone.
- SCALE event - normal SCALE events, mostly Sat/Sun.
- S.T.A.R. event - Special Tracks Are Restricted. e.g. SCALE University
  or a law class. In case an attendee gets confused and wants in with a
  SCALE pass, we can explain to them what S.T.A.R stands for. :)

General policy changes / clarifications:

- Hosted events must go through SCALE registration, to avoid confusion
  where people register with some other event, and then can't find
  themselves in SCALE's registration DB.
- To alleviate the need for separate registration, hosted events can
  become Limited events. We will share Limited event registration info
  with the event. Attendees will be notified.
- Kids passes (up to 4) are free for anyone under the age of 18 with a
  paid adult pass. This helps to keep the show family friendly.

To explain ticket changes in detail:

- Expo Pass - dropped "only" because it allows access to non-limited
  hosted events too. Price goes up to $20 to make it easier if they walk
  in and pay cash. Not coupon eligable.
- Limited Pass - gives access to a limited event. May cost more if the
  limited event has their own fees. Registration information will be
  shared with the event. Doubles as an Expo Pass. Not coupon eligable.
- SCALE Pass - renamed from "full access" pass to avoid confusion
  because some limited / S.T.A.R. events are off limits. Dobules as an
  Expo pass. Cash prices are now in multiples of $20. Sunday only pass,
  which we sell very few of, are now $40 instead of $45.
- Supporter Pass - copied from TXLF. $100 even. We will put the names of
  the supporters that register early enough in the program in the credit
  section next to the chairs/volunteers. [Other ideas welcome]
- S.T.A.R. Pass - Because I didn't care for gold/platinum/diamond.

Once we agree on these changes, I'd like to publish this info to the
SCALE 15X website before registration opens and link to it from
registration when it opens.

For reference, 14X prices were:

Expo only: $15
Full (early): $75
Full (normal): $85
Full (late): $90
Full (sunday): $45
Law: $150

As a random point of comparison, the LA Auto Show tickets are $12 / day
weekdays and $15 / day Friday + weekend. For that price, you get one day
to walk around the LACC, look at new cars, and sit in some of them. No,
Porsche and Masarati don't let attendees sit in their cars or even touch
them.

**Comment by irabinovitch** [comment]
You can see a mock up of this pricing at: http://leiz.org/scale/tickets.html

**Comment by irabinovitch** [comment]
Did anyone have any feedback on this one?

**Comment by davidelang** [comment]
this hardly seems like a simplification

**Comment by jillr** [comment]
I agree with davidelang.  It's unclear to me how registration for multiple events would work.  If I want a SCALE pass and Ubucon for example, would those need to be separate transactions in the reg system or are colocated events "add-ons" to SCALE reg?  Also, have we solicited feedback from the usual collocated events?

**Comment by mrsiltanen** [comment]
1) I REALLY LIKE the discounted pass idea for Sunday, imho it is critical we keep Sunday a strong and vibrant day.

2) Does the discount codes apply to all these?? 

3) Confusion about what is covered in the costs often is a question of alignment / clarification on the website, the schedule, and ticket registration page.
imho it is often takes more work to determine what requires additional registration / fees. I've had some challenges explaining this to the various community groups I've gone to, so I'd understand if someone new to SCALE would also. The solution is probably to have a UX/UI check with a non-SCALE test group.

4) If I pay onsite with CC / check does that mean it's a ( late ) pass?

5) Note: Charging a higher price for those using cash instead of CC further disadvantages those who are underbanked. Thus I would recommend against it if we are wanting to be more diverse and inclusive.

6) Like the kids free w/paid adult.

Thank You
Matti

**Comment by irabinovitch** [comment]
@jillr This confusion is in part one of the issues we're trying to address. In the past it's been unclear because groups like UbuCon and PuppetCamp have run their own events outside of SCALE registration. Sometimes these were free, sometimes they were paid. This left attendees confused as to what was included in their passes and whether or not they had to pay for SCALE and the other event.  Do you have thoughts on how to better phrase or break this down?

**Comment by irabinovitch** [comment]
In terms of Sunday only passes, I'm generally opposed to 1 day tickets.  Sunday only *might* be ok in that Sunday is low traffic and we'd only allow purchasing of it the day of in cash if I'm reading this correctly?

**Comment by davidelang** [comment]
On Mon, 18 Jul 2016, Ilan Rabinovitch wrote:

> @jillr This confusion is in part one of the issues we're trying to address. In 
> the past it's been unclear because groups like UbuCon and PuppetCamp have run 
> their own events outside of SCALE registration. Sometimes these were free, 
> sometimes they were paid. This left attendees confused as to what was included 
> in their passes and whether or not they had to pay for SCALE and the other 
> event.  Do you have thoughts on how to better phrase or break this down?

I think the obvious thing is that if Scale is hosting it, you have to pay the 
Scale registration. If an event wants to collect money above and beyond that, it 
would be a separate fee.

Anyone who is there has access to the 'standard' scale events/talks, and we 
should discourage extra fee sub-events, but support them if it's appropriate to 
the Scale mission.

David Lang

**Comment by jillr** [comment]
@irabinovitch A single-select SCALE registration option (Full, Sunday-only, expo-only, etc) and then a set of multi-select 'add-on' (or some similar verbiage) Partner (or maybe Community?) events.  So I might pick Full SCALE Reg then add-on PuppetCamp and DevOps Day.  Then they would all be line-item'ed on my one registration receipt, either at $0.00 or $some_money.  This would also help people that get reimbursement for some or all events.  

But then, if someone wants to do a colocated event but no SCALE events there would need to be a way for them to select something like "No SCALE - I only want to attend a colocated event" and that could cause additional confusion of a different type.  

There would also need to be a way to smoosh all of this info onto one badge, maybe using logos for colocated events.  Is there enough room on the current badge template if someone were to hypothetically register for all colocated events, full SCALE pass, and add a gpg key?

**Comment by davidelang** [comment]
On Mon, 18 Jul 2016, Jill R wrote:

> @irabinovitch A single-select SCALE registration option (Full, Sunday-only, 
> expo-only, etc) and then a set of multi-select 'add-on' (or some similar 
> verbiage) Partner (or maybe Community?) events.  So I might pick Full SCALE 
> Reg then add-on PuppetCamp and DevOps Day.  Then they would all be 
> line-item'ed on my one registration receipt, either at $0.00 or $some_money. 
> This would also help people that get reimbursement for some or all events.
>
> But then, if someone wants to do a colocated event but no SCALE events there 
> would need to be a way for them to select something like "No SCALE - I only 
> want to attend a colocated event" and that could cause additional confusion of 
> a different type.

I'm not sure we should encourage/support this.

If the attendees of an event are not going to be allowed to participate in 
anything Scale related, why hold it the same place the same day?

> There would also need to be a way to smoosh all of this info onto one badge, 
> maybe using logos for colocated events.  Is there enough room on the current 
> badge template if someone were to hypothetically register for all colocated 
> events, full SCALE pass, and add a gpg key?

There should not be a lot of paid add-ons, and there should be an upper limit to 
the cost (a 'gold pass' so to speak) that lets someone wander in and out of 
anything. they aren't going to get the 'full value' of any of them, but it lets 
them get what they want out of what's offered.

This is open source, where we take bits and pieces from everything and combine 
them in unique ways. Let's avoid making the registration be rigid partitioning 
of people into different groups.

David Lang

**Comment by jillr** [comment]
I would also note, as an attendee I've never found too much confusion in registering for cohabiting events separately, and do think there's merit in letting those events continue to handle their own registration (and therefore attendee lists, fees, updates/communications to registrants, etc) but if we were to move forward with collapsing registration for everything into scale reg, those would be my comments/concerns.

**Comment by irabinovitch** [comment]
@jillr I think we generally to just get rid of the "no SCALE option" and have all the reg run through SCALE.  And ya I dont know how to represent it all on a badge yet. Ill defer to @leizleiz and @ScislaC on that once we figure out the levels we want to offer

**Comment by irabinovitch** [comment]
@davidelang Thanks for the feedback David, but given we let other groups run events at SCALE we can't always control their costs or what the upper limits are.  Eg we have to charge extra to track CLE credits for attorney's "gold" pass or otherwise.

**Comment by jillr** [comment]
@irabinovitch gotcha.  For clarity, what are the (recent, historical) paid addons?  I know Elastic has done paid training, and the Legal, but I can't think anything else offhand.  So how would this affect things like 'vendors offering paid training/certifications onsite'?  Those would now need to go thru scale reg and that's the crux of the change?

**Comment by davidelang** [comment]
On Mon, 18 Jul 2016, Ilan Rabinovitch wrote:

> @davidelang Thanks for the feedback David, but given we let other groups run 
> events at SCALE we can't always control their costs or what the upper limits 
> are.  Eg we have to charge extra to track CLE credits for attorney's "gold" 
> pass or otherwise.

official credits for various purposes are one valid exception to the 'gold pass' 
cap, but they also don't need to be represented on the badge in any way.

David Lang

**Comment by davidelang** [comment]
On Mon, 18 Jul 2016, Ilan Rabinovitch wrote:

> @davidelang Thanks for the feedback David, but given we let other groups run 
> events at SCALE we can't always control their costs or what the upper limits 
> are.  Eg we have to charge extra to track CLE credits for attorney's "gold" 
> pass or otherwise.

I also think it should be possible to take the class and not get the CLE (or 
whatever) credits, and get a discounted price.

David Lang

**Comment by jillr** [comment]
Assuming colocated events that are paid (like trainings) even all agreed to pass their payments through the SCALE system (and I'd assume some wouldn't), that also means SCALE has inserted itself into the process for refunds/disputes for these events.  And has taken on a great deal more accounting.  Is a single reg system for all events worth that trade off?

**Comment by mrsiltanen** [comment]
Hi,
I'd like to see as much as possible included in the FULL SCALE pass.

Remember we do not currently have door people ( volunteers or otherwise ) checking passes for the various rooms, which I think makes our event friendlier than other events. ( and saves on some big headaches I have seen in other events )

Thank YouMatti

 

    On Monday, July 18, 2016 5:50 PM, Jill R <notifications@github.com> wrote:
 

 @irabinovitch gotcha. For clarity, what are the (recent, historical) paid addons? I know Elastic has done paid training, and the Legal, but I can't think anything else offhand. So how would this affect things like 'vendors offering paid training/certifications onsite'? Those would now need to go thru scale reg and that's the crux of the change?—

**Comment by irabinovitch** [comment]
On Mon, Jul 18, 2016 at 5:52 PM, davidelang <notifications@github.com>
wrote:

> On Mon, 18 Jul 2016, Ilan Rabinovitch wrote:
>
> > @davidelang Thanks for the feedback David, but given we let other groups
> run
> > events at SCALE we can't always control their costs or what the upper
> limits
> > are. Eg we have to charge extra to track CLE credits for attorney's
> "gold"
> > pass or otherwise.
>
> I also think it should be possible to take the class and not get the CLE
> (or
> whatever) credits, and get a discounted price.


Thats under discussion with the CA bar association.  Different discussion
for a different day.

**Comment by irabinovitch** [comment]
On Mon, Jul 18, 2016 at 5:58 PM, mrsiltanen <notifications@github.com>
wrote:

> Remember we do not currently have door people ( volunteers or otherwise )
> checking passes for the various rooms, which I think makes our event
> friendlier than other events. ( and saves on some big headaches I have seen
> in other events )
>

We've generally followed an honor system, and I dont see this changing
anytime soon.

**Comment by lufthans** [comment]
It's great to have all passes go through SCaLE to simplify registration for attendees.

Is there an icon we can add to denote options that include a full SCALE pass? If we had that ( or the opposite denoting when it is not included ), then we could list the SCALE pass first. Presuming that's how most people register, it's good to have it be the first option.

How will the various prices and deadlines for the SCALE pass be communicated? Ho
w about "| SCALE Pass (early bird pricing ends on YYYYMonDD) | $75 |" with infor
mation below about early bird and late pricing.

The 1 to 5 char promo code only applies to SCALE pass? Whatever it applies to sh
ould be mentioned.

I love the no cost for kids with paid adult option.

I agree with others that the cash price should not be more than CC price. Do we offer promo code discounts on late and cash pricing?

Supporter pass is a great idea.

Is group pricing handled via the short promo code? Should the registration page mention group pricing? Do we want to restrict that type of pricing to school/children groups such as Maker Scouts and school groups?

**Comment by leizleiz** [comment]
We ended up with the following for SCALE 15X:
- 3 base ticket types: Expo, SCALE, and Limited passes.
- Better explanation of what the passes are on the registration page.
- Blurb to explain promo codes generally do not apply to Expo passes and Limited passes.
- The only thing not included in the SCALE pass is the Open Source Legal Training.
- The PR team is sending out announcements before each price increase.
- I lowered the cash pricing to the same price as the credit card option on the day of. I will just go to the bank and get some extra change, as annoying as that is.

Ilan has been handling group sales. I don't know how groups contacted him, but maybe we can add more info for group sales on the registration page.
